### PR TITLE
Sync latest changes from GLEIF-IT/development excluding GLEIF branding and witness pools

### PR DESCRIPTION
**Pull request by kentbull**
This includes all of the latest changes from GLEIF-IT/development with the exception of the GLEIF branding and witness pools.

### CODE REVIEW

**Review by pfeairheller — DISMISSED** [review]
Can't hurt, right?

File: app/assets/gleif.png
Comment (m00sey) [inline app/assets/gleif.png:1]: this should not be included

File: app/assets/icon.png
Comment (m00sey) [inline app/assets/icon.png:1]: this should not be included

Comment (kentbull) [reply app/assets/icon.png:1]: Thanks for catching that. I removed two other GLEIF images as well from the src/assets/img/logo folder.

File: app/assets/gleif.png
Comment (kentbull) [reply app/assets/gleif.png:1]: removed in the latest commit

**Review by m00sey — APPROVED** [review]
lgtm

**Pull request merged**